SmartDeviceLink-iOS/SmartDeviceLink/SDLProtocol.m

```diff
@@ -28,6 +28,7 @@ @interface SDLProtocol () {
     dispatch_queue_t _sendQueue;
     SDLPrioritizedObjectCollection *_prioritizedCollection;
     NSMutableDictionary *_sessionIDs;
+    NSMutableDictionary *_hashIDs;
```

Comment:
This would probably be a good idea for a future enhancement. We would need to make sure it's dynamic enough to accept any future metadata. Would this be suitable for now, do you think?